Survey/Sources/Presentation/Modules/Home/HomeView.swift

```diff
@@ -25,24 +25,22 @@ struct HomeView: View {
             text: .constant(""),
             content: { geo in
                 ZStack(alignment: .top) {
-                    ScrollView(.horizontal) {
-                        LazyHStack {
-                            tabViewSetup()
-                                .frame(
-                                    width: geo.size.width,
-                                    height: geo.size.height
-                                )
-                        }
-                        .overlay(alignment: .top) {
-                            HeaderHomeView()
-                                .padding(.top, 60.0)
-                                .padding(.leading, 20.0)
-                        }
+                    LazyHStack {
+                        tabViewSetup(geo: geo)
+                            .frame(
+                                width: geo.size.width,
```

Comment:
Apart from login screen layout, we don't seem to use GeoReader for anything else. I think it will take more resource from the device if we use GeoReader as default, have to add more lines to all views, and could create bugs in the long run.
I suggest moving GeoReader from LoadingView and use it only for wrapping LoginView's ZStack only for now.
Please note that you will need to modify some layout in `SurveyItemView` to accommodate small devices.
What do you think?

Reply:
Really thank you 🙏  for your suggestion.

At first place, I use **GeometryReader**  for `LoadingView` that could give me the dimensions of entire screen (I put the **GeometryReader** in my first view hierarchy) and some advantage came with **GeometryReader**, so I think it's fine to have it 😄. But as your great analysis 👍 about disadvantages on **GeometryReader** and I totally agree with that 🙇 and I removed **GeometryReader** from `LoadingView` at https://github.com/nkhanh44/ic-survey-ios/pull/18/commits/0093261ceda17685a647a3db1ade5c330fac9868

I also tested on smaller devices, that works well 👍.

Reply:
Looks good. Thank you. One last comment. https://github.com/nkhanh44/ic-survey-ios/commit/0093261ceda17685a647a3db1ade5c330fac9868#comments